proto/import_sstpb.proto

```diff
@@ -185,6 +185,11 @@ message SetDownloadSpeedLimitResponse {
 message Pair {
     bytes key = 1;
     bytes value = 2;
+    enum OP {
+        Put = 0;
+        Delete = 1;
```

Comment:
Tag 0 in enum is the default value. It's better to reserve the tag for Unknown to avoid surprising behavior.
```suggestion
        Unknown = 0;
        Put = 1;
        Delete = 2;
```

Reply:
I think Put = 0 is better, because lightning local backend already use this protocol. we can avoid incompatible during upgrade.